Oswaldo Vazquez Aparicio/26 Agosto/miniproject/ShoppingList.java

```diff
@@ -0,0 +1,30 @@
+package miniproject;
+
+import java.util.LinkedList;
+
+public class ShoppingList {
+
+    private final LinkedList<Product> products;
+
+    public ShoppingList(){
+        products = new LinkedList<>();
+    }
+
+    public void addProductToBuy(Product product){
+        products.add(product);
+    }
+
+    public void printProducts(){
+        String tableFormat = "%-30s%-15s%-15s%-15s%n";
+        String separatorLine = "========================================================================"
+        System.out.println(separatorLine);
+        System.out.printf(tableFormat, "Name", "UnitPrice", "Quantity", "Total Price");
+        System.out.println(separatorLine);
+        products.forEach(product -> {
```

Comment:
Nice job using streams forEach... what if products is empty???

Reply:
There isn't gonna print any elements, it just gonna show an empty table. Should I print a message instead of empty elements?